sns/Dependencies.toml

```diff
@@ -0,0 +1,310 @@
+# AUTO-GENERATED FILE. DO NOT MODIFY.
```

Comment:
Let's not add the auto generated Dependencies.toml

We'll add the Dependencies.toml like we have in other handwritten connectors